scarb/src/bin/scarb/args.rs

```diff
@@ -341,6 +341,9 @@ pub struct FmtArgs {
     /// Specify package(s) to format.
     #[command(flatten)]
     pub packages_filter: PackagesFilter,
+    /// Path to a file or directory to format. If provided, only this file or directory will be formatted.
+    #[clap(value_name = "PATH")]
+    pub target_path: Option<Utf8PathBuf>,
```

Comment:
I would seriously consider avoiding the word `target`. I think `--path` would be just fine here, wdyt?

Reply:
exactly, thx